src/dotnet/Notification.Service/Db/DbChatSubscription.cs

```diff
@@ -0,0 +1,22 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Stl.Versioning;
+using Microsoft.EntityFrameworkCore;
+
+namespace ActualChat.Notification.Db;
+
+[Table("ChatSubscriptions")]
+[Index(nameof(UserId), nameof(ChatId))]
+public class DbChatSubscription : IHasId<string>, IHasVersion<long>
```

Comment:
IMO not good... We won't be able to shard this assuming there is `DbDevice` & `DbMessage` too.

Reply:
I would probably move this table to Chats (access it via `IChats` or create dedicated `IChatNotifications` & store this stuff there).